examples/bezier/app.js

```diff
@@ -0,0 +1,115 @@
+/* global window,document */
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+import DeckGLOverlay from './deckgl-overlay.js';
+
+import SAMPLE_GRAPH from './sample-graph';
+
+function computeControlPoint(source, target, direction, offset) {
```

Comment:
shall we add jsdoc or a link to some reference to explain how to use the direction and offset here?